var/spack/repos/builtin/packages/thepeg/package.py

```diff
@@ -71,13 +71,15 @@ class Thepeg(AutotoolsPackage):
     depends_on('automake', type='build')
     depends_on('libtool',  type='build')
     depends_on('m4',       type='build')
+    depends_on('zlib',     type='build')
```

Comment:
I think the package also links to `zlib`:
```suggestion
    depends_on('zlib')
```